src/internal.c

```diff
@@ -6309,7 +6309,7 @@ static void InitSuites_EitherSide(Suites* suites, ProtocolVersion pv, int keySz,
         word16 haveFalconSig, word16 haveDilithiumSig, word16 haveAnon,
         int side)
 {
-    /* make sure server has DH parms, and add PSK if there */
+    /* make sure server has DH paras, and add PSK if there */
```

Comment:
believe that this should be params (short for parameters)

Reply:
Of course, that's a typo "fixed" by another typo :smile: